etor/src/main/resources/openapi_etor.yaml

```diff
@@ -74,6 +74,43 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/UnauthorizedRequestError'
+  /v1/etor/metadata/{id}:
+    get:
+      summary: Returns metadata about a specific submission
+      parameters:
+        - name: id
+          in: path
+          description: A valid id for the submission metadata
+          required: true
+          schema:
+            type: string
+      security:
+        - bearerAuthorization: [ ]
+      responses:
+        '200':
+          description: The metadata was returned successfully
+          content:
+            application/json:
+            schema:
+              $ref: '#/components/schemas/MetadataResponse'
+        '401':
+          description: Authentication failed due to invalid token or unknown organization
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedRequestError'
+        '404':
+          description: Unable to find requested metadata
+          content:
+            application/json:
+            schema:
+              $ref: '#/components/schemas/NotFoundError'
+        '500':
+          description: Unable to retrieve requested metadata
+          content:
+            application/json:
+            schema:
+              $ref: '#/components/schemas/InternalServerError'
```

Comment:
I wouldn't include `500`.  That's when something goes big boom and generally isn't expected given any input from a user.